docs/api/server/CLI-Commands.md

```diff
@@ -0,0 +1,204 @@
+```
+
+## Start Commands
+
+| Command       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `start`  | Runs `one-app` Server and accepts the [flags listed below](#start-usage). |
+| `start:watch` | Executes `one-app` Server and watches Server source files with [`nodemon`](https://nodemon.io/). When source changes, `one-app` reloads with the `npm start`. This command [accepts all the flags as `start`](#start-usage). |
+
+### Start Usage
+
+```sh
+SOME_ENV_VAR=value npm run start -- <flags>
+# e.g. ONE_CONFIG_ENV=development npm run start -- --root-module-name=my-root-module
+```
+
+#### Flags
+
+| Flag       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `--root-module-name`  | Name of the [Holocron Module] that serves as the entry point to your application. |
+| `--module-map-url`  | Remote Module Map URL for [`one-app-dev-cdn`] to proxy. *For Development.* |
+| `--use-middleware` | Apply a custom middleware configuration for [`one-app-dev-proxy`]. *For Development.* |
+
+#### Environment Variables
+
+Please see the [Environment Variables](./Environment-Variables.md) API docs.
+
+## Test Commands
+
+| Command       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `test` | Runs all `test:*` scripts. |
+| `test:unit` | Runs all [Jest] Unit Tests. Accepts Jest flags. |
+| `test:lint` | Runs [ESLint] [rules](https://github.com/americanexpress/eslint-config-amex) against source. |
+| `pretest:integration` | Using [`concurrently`], runs `build:prod-sample` and pulls Docker containers to prepare the [Integration Test Suite]. |
+| `test:integration` | Executes the [Integration Test Suite] using [Jest]. |
+| `test:danger` | Runs the [DangerJS tool](https://danger.systems/js/) that returns bundle size stats and a preflight checklist for PRs. |
+| `test:git-history` | Validates the Git commit message format using the [`commitlint`](https://github.com/conventional-changelog/commitlint) tool. |
+| `test:lockfile` | Validates that the `package-lock.json` file contains valid NPM registry URLs using the [`lockfile-lint`](https://github.com/lirantal/lockfile-lint) tool. |
+| `prestart:prod-sample` | Runs [`build:sample-modules`](#build-commands) before `start:prod-sample`. |
+| `start:prod-sample` | Runs a [Docker Compose](https://docs.docker.com/compose/compose-file/) file to start up the [Integration Test Suite] containers. |
+
+## Development Commands
+
+> 💬 These commands only work in Development
+
+| Flag | Description |
```

Comment:
these arent flags

Reply:
Arguments?

Reply:
Fixed in a88378aafc426e868db005159f4e769323b6a93d